readme.md

```diff
@@ -35,6 +35,16 @@ tempy.directory({prefix: 'name'});
 
 Get a temporary file path you can write to.
 
+### tempy.file.task(callback, options?)
+
+The `callback` resolves with a temporary file path you can write to. The file is automatically cleaned up after the callback is executed.
+
+#### callback
+
+Type: `(tempPath: string) => any`
```

Comment:
Like commented elsewhere. Should be `void`. (When I comment on something, it's smart to look for other places it applies too).

Reply:
Done!